xfields/beam_elements/beambeam3d.py

```diff
@@ -68,8 +98,20 @@ class BeamBeamBiGaussian3D(xt.BeamElement):
         'min_sigma_diff': xo.Float64,
         'threshold_singular': xo.Float64,
 
+        # beamstrahlung 
+        'flag_beamstrahlung': xo.Int64,
+        'slices_other_beam_zeta_bin_width_star': xo.Float64[:],
+
+        # luminosity
+        'flag_luminosity': xo.Int64,
+
+        # for debugging
+        "turn": xo.Int64,
+        #"flat_y_i": xo.Float64,
```

Comment:
Remember to remove these? Is flag_luminosity also going away?

Reply:
turn and flat_y_i were there just for me for some debugging. they will go away. Also the lumi table will go away for now. Later I might have to add it for the Bhabha scattering.